modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/RuntimeProvisioningService.java

```diff
@@ -23,45 +23,177 @@
 
 public interface RuntimeProvisioningService {
 
+    /**
+     * Set application details for the context
```

Comment:
A javadoc comment should end with the period. Else this will throw error in a windows build. Fix these type of javadoc issues on all classes.